match/lib/match/options.rb

```diff
@@ -64,6 +64,16 @@ def self.available_options
                                      verify_block: proc do |value|
                                        ENV["FASTLANE_TEAM_ID"] = value.to_s
                                      end),
+        FastlaneCore::ConfigItem.new(key: :git_user_name,
+                                     env_name: "MATCH_GIT_USER_NAME",
+                                     description: "git user name to commit",
```

Comment:
Maybe that's just a small detail, but by `user name` many people understand the nickname (in my case `KrauseFx`), however in git it's recommended to use the full name as far as I know (in my case `Felix Krause`). Maybe we should rename the `key`, `env_name` and `description` to clarify this?